src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1679,9 +1681,29 @@ public class ClusterVNode<TStreamId> :
 					_mainQueue.Publish(new AuthenticationMessage.AuthenticationProviderInitializationFailed());
 				} else {
 					_mainQueue.Publish(new AuthenticationMessage.AuthenticationProviderInitialized());
+					_mainQueue.Publish(TimerMessage.Schedule.Create(TimeSpan.FromHours(24),
+						new PublishEnvelope(_mainQueue),
+						new SystemMessage.VerifyNodeCertificateExpiry()));
+
 				}
 			});
 		}
+		
+		public void Handle(SystemMessage.VerifyNodeCertificateExpiry message) {
+			var (certificate, intermediates) = _options.LoadNodeCertificate();
```

Comment:
Using `LoadNodeCertificate()` here causes an error to be thrown when running in either `--dev` or `--insecure` mode because there aren't any certificate options set for either of those cases:

```
[12880,15,13:32:06.971,ERR] Error while processing message "EventStore.Core.Messages.SystemMessage+VerifyNodeCertificateExpiry" in queued handler '"MainQueue"'.
EventStore.Common.Exceptions.InvalidConfigurationException: A certificate is required unless insecure mode (--insecure) is set.
   at EventStore.Core.ClusterVNodeOptionsExtensions.LoadNodeCertificate(ClusterVNodeOptions options) in C:\Users\hayle\code\EventStore\src\EventStore.Core\ClusterVNodeOptionsExtensions.cs:line 243
   at EventStore.Core.ClusterVNode`1.Handle(VerifyNodeCertificateExpiry message) in C:\Users\hayle\code\EventStore\src\EventStore.Core\ClusterVNode.cs:line 1693
   at EventStore.Core.Bus.MessageHandler`1.TryHandle(Message message) in C:\Users\hayle\code\EventStore\src\EventStore.Core\Bus\MessageHandler.cs:line 30
   at EventStore.Core.Bus.InMemoryBus.Publish(Message message) in C:\Users\hayle\code\EventStore\src\EventStore.Core\Bus\InMemoryBus.cs:line 290
   at EventStore.Core.Bus.QueuedHandlerMRES.ReadFromQueue(Object o) in C:\Users\hayle\code\EventStore\src\EventStore.Core\Bus\QueuedHandlerMRES.cs:line 124
[12880,25,13:33:49.018,INF] Exiting with exit code: 0.
```

Instead, it would be better to use the `_certificateSelector` to get the certificate, as this will try and get the cert from the correct provider (either from options, or from dev certificates)

```suggestion
			var certificate = _certificateSelector();
```

You will need to do a null check on the certificate after getting it though, because it will always be null in `--insecure` mode.